presto-server-rpm/pom.xml

```diff
@@ -170,6 +170,14 @@
                                     </includes>
                                 </rule>
 
+                                <rule>
+                                    <destination>/etc/presto/catalog</destination>
```

Comment:
Rather than adding a new entry for `/etc/presto/catalog`, can we change the one above for `/etc/presto` to be the following:
```xml
<include>*/**</include
```
(untested -- based on the one used for `/usr/lib/presto/bin`)